src/haven/plans/robot_transfer_sample.py

```diff
@@ -0,0 +1,97 @@
+import logging
+
+from bluesky import plan_stubs as bps
+
+from ..instrument.instrument_registry import registry
+from ..motor_position import rbv
+
+log = logging.getLogger(__name__)
+
+
+__all__ = ["robot_transfer_sample"]
+
+
+def robot_transfer_sample(robot, sampleN, *args):
+    """
+    Use robot to load sampleN at a fixed Aerotech stage position or any
+    motors.
+
+    e.g.
+    robot_sample(robot, 9, motor1, 100, motor2, 200, motor3, 50)
+
+    Parameters:
+    - robot: robot device
+    - sampleN: Sample number
```

Comment:
We should include something in the docstring to let clients know that ``sampleN=None`` will unload the current sample.